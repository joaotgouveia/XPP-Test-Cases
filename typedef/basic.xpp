type my_type = int;
(
let f = fn x: my_type => { x }; f(1)
);;
