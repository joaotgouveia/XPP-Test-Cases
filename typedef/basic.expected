Value: 1
Type: int
